.env.sample

```diff
@@ -1,3 +1,4 @@
 AIRTABLE_API_KEY=YOUR_KEY_HERE
+AIRTABLE_BASE_ID=appNYMWxGF1jMaf5V
```

Comment:
Guessing it allows us to easily switch to a staging base, which we were both doing for testing reasons.